en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md

```diff
@@ -665,7 +766,7 @@ This section involves setting up the Gateway node and enabling it to work with t
     password = "$ref{super_admin.password}"
     ```
 
-4.  If you need to enable JSON Web Token (JWT), you have to enable it in all Gateway and Key Manager components.
+4.  If you need to enable JSON Web Token (JWT), you have to enable it in all Gateway components.
```

Comment:
in all Gateway components - > in all the gateway components.
gateway components? or gateway nodes?